src/main/java/edu/kis/powp/jobs2d/drivers/factory/FiguresFactory.java

```diff
@@ -0,0 +1,75 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.jobs2d.drivers.command.ComplexCommand;
+import edu.kis.powp.jobs2d.drivers.command.DriverCommand;
+import edu.kis.powp.jobs2d.drivers.command.OperateToCommand;
+import edu.kis.powp.jobs2d.drivers.command.SetPositionCommand;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FiguresFactory {
+
+    public enum Figure {
+        RECTANGLE,
+        TRIANGLE
+    }
+
+    private static class Rectangle {
+        public static final int x1 = 200;
+        public static final int y1 = 200;
+
+        public static final int x2 = 200;
+        public static final int y2 = -200;
+
+        public static final int x3 = -200;
+        public static final int y3 = -200;
+
+        public static final int x4 = -200;
+        public static final int y4 = 200;
```

Comment:
Plagiarism! :(

Reply:
Expanded FiguresFactory to be more generic. 
Available configurable shapes: square, triangle, trapezium, cross, arrowTip.
Added multiple tests to TestJobs2dPatterns -> some shapes in multiple versions to present flexibility of the implementation.